test/docker/compose.go

```diff
@@ -291,6 +291,7 @@ func (d *Compose) WithBasicAuth(username, password string) *Compose {
 }
 
 func (d *Compose) WithWeaviateAuth() *Compose {
+	d.withWeaviateAuth = true
```

Comment:
This is present on master and probably got lost on refactor in previous commits. Caused a failure with auth tests in go client tests.